samples/bookinfo/kube/mixer-rule-ratings-ratelimit.yaml

```diff
@@ -49,3 +49,28 @@ spec:
   - handler: handler.memquota
     instances:
     - requestcount.quota
+---
+apiVersion: config.istio.io/v1alpha2
+kind: QuotaSpec
+metadata:
+  creationTimestamp: null
+  name: request-count
+  namespace: istio-system
+spec:
+  rules:
+  - quotas:
+    - charge: 1
+      quota: RequestCount
+---
+apiVersion: config.istio.io/v1alpha2
+kind: QuotaSpecBinding
+metadata:
+  creationTimestamp: null
+  name: request-count
+  namespace: istio-system
+spec:
+  quotaSpecs:
+  - name: request-count
```

Comment:
`namespace: istio-system` is needed here.

Reply:
Same namespace can be elided in mixer config. Perhaps it should be here too.
This is passing all the tests though...

Reply:
oops, sorry. @ayj do you want to revert this or check in another change with `namespace`